index.html

```diff
@@ -0,0 +1,1280 @@
+                                <img src="https://d.newsweek.com/en/full/1559216/tokyo-my-city-chizu-nakamoto.jpg?w=500&h=333&f=9c0c7599e1d2310c5649861e93649619" alt="">
+                        </div>
+                        <div class="art-cnt s-insight-cnt">
+                                <h4> Female Entrepreneur Active in Tokyo and Uganda </h4>
+                                <p> APT Women aims to foster and educate fledgling global business leaders like Chizu Nakamoto. </p>
+                        </div>
+                        
+                    </article> 
+
+                    <article class="art s-insight">
+                        <div class="art-img">
+                            <div class="img-caption blue-text"> SPONSORED INSIGHT </div>
+                                <img src="https://d.newsweek.com/en/full/1573613/university-birmingham-dubai.jpg?w=500&h=333&f=230990a1f18e561a516f2f98ebfda59f" alt="">
+                        </div>
+                        <div class="art-cnt s-insight-cnt">
+                                <h4> How Can a Good Business Education Benefit You? </h4>
+                                <p> Enrolling at a good Business School might be the best investment you can make in your future, and while costs can be significant, the rewards are incalculable. Find out more. </p>
+                        </div>
+                        
+                    </article> 
+
+                </div>
+
+
+            </div>
+            </div>
+        </div>
+        
+
+        <div class="p-ctn container-fluid">
+
+            <div class="mag-title d-flex justify-content-center">
+            IN THE MAGAZINE
+            </div>
+
+            <div class="row in-mag-row1"> 
+                    <article class="art art-feat col-md-3">
+                        <div class="art-img">
+                            <div class="img-caption"> U.S.</div>                        
+                            <img src="https://d.newsweek.com/en/full/1576902/april-03-2020-cover.jpg?w=360&h=480&l=0&t=0&q=90&f=32baa29597300c7a9e09b35b8d6b1265" alt="">
+                        </div>
+
+                        <div class="ad-title">
+                        see all features &nbsp;>
+                        </div>
+                    </article> 
+
+                    <article class="art art-feat col-md-3">
+                        <div class="mag-img">
+                                <img src="https://d.newsweek.com/en/full/1575272/fe-covid-cure-opener.jpg?w=466&h=311&l=47&t=40&q=90&f=7d5d8ed57150fd6695e0e7d465be9cf5" alt="">
+                        </div>
+                        <div class="mag-caption"> <span class="m-caption"> Feature</span> <span class="mag-tag"> Tech and Science </span> </div>
+                        <div class="mag-cnt">
+                                <h4> Is chloroquine the Answer to COVID-19 Pandemic? </h4>
```

Comment:
The images in this section have the correct size when the screen is larger than 960px, but they look disproportioned. 

Then, when you go below 960, the proportion is correct, but the size is not (it should have a larger height). 

Finally, when you are on mobile, the margins are too large compared to the original page.

Please make the necessary adjustments to make them look like the original page.